### PR TITLE
Handlers

### PR DESCRIPTION
**Pull request by BillyAddlers**
- Adding Command Handlers using custom CommandClient. Using similar concept as [DiscordJS-Commando](https://github.com/discordjs/Commando) and [DiscordJS-Akairo](https://github.com/discord-akairo/discord-akairo). Extensible with Abstract Class that will extended into a prefered commands. Fully configurable.

- Adding few rules so that the ESLint won't bother much. Still not perfect, though. Recommended alternative: [Standard](https://github.com/standard/standard).

- Adding new Logging System, implemented above the old Logging System: LoggingFactory (it should be LoggerFactory, but meh. Whatever).

Command Example: 
awawa

TODO: Event Handler, Database, Mocha Test Unit.

> note: I recommend removing package-lock.json from repository, as it causes errors with Yarn Package Manager.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -28,6 +28,243 @@ module.exports = {
         }
       }
     ],
+    rules: {
```
Comment (dwicao) [inline .eslintrc.js]: is this correct? you've just added an `rules` object inside `rules`.
like so `rules.rules.curly`
it should be just `rules.curly`

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,10 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# Lockfiles
+package-lock.json
```
Comment (dwicao) [inline .gitignore]: this is already defined at https://github.com/japanisme/japanisme-bot/pull/10/files#diff-a084b794bc0759e7a6b77810e01874f2R71

so please delete it

File: .eslintrc.js (hunk 1 above)
Comment (BillyAddlers) [reply .eslintrc.js]: Whoops. My bad, sowwy. Feel free to edit or remove any code that you think it's absolutely wrong or causes error.

File: src/config.json [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+        "327586923252285440", 
+        "442830299781529610", 
+        "569874192733044747",
+        "510749266499731467", 
+        "369507341973979136",
+        "337028800929857536"],
+
+    "dev_prefix": "_",
+    "prefix": "a!"
+}
```
Comment (dwicao) [inline src/config.json]: the `owners` object is already defined,
can you please delete the unused one?

Comment (SharifPoetra) [reply src/config.json]: I'll do it

File: .gitignore (hunk 2 above)
Comment (SharifPoetra) [reply .gitignore]: okay

File: src/main.js [hunk 4]
```diff
@@ -1,3 +1,8 @@
+// const { Japanisme } = require("./handler/client");
```
Comment (dwicao) [inline src/main.js]: please delete this unused code

Comment (SharifPoetra) [reply src/main.js]: Already on my work👍🏻

**Pull request merged**